elegant/clean_timepoint_data.py

```diff
@@ -113,11 +122,30 @@ def remove_excluded_positions(experiment_root,dry_run=False):
     annotations = load_data.read_annotations(experiment_root)
     good_annotations = load_data.filter_annotations(annotations, load_data.filter_excluded)
     excluded_positions = sorted(set(annotations.keys()).difference(set(good_annotations.keys())))
+
     for position in excluded_positions:
         if (experiment_root / position).exists():
             print(f'Found an excluded position to delete {position}')
             if not dry_run:
+                (experiment_root / 'excluded_positions' / position).mkdir(parents=True,exist_ok=True)
+                (experiment_root / 'excluded_positions' / 'annotations').mkdir(parents=True,exist_ok=True)
+
+                shutil.copy(str(experiment_root / position / 'position_metadata.json'),
```

Comment:
As of at least Python 3.6 (if not earlier?) `shutil` can ingest `pathlib.Path` objects directly, without stringifying them. (In at 3.6, it should be a bug for any python standard library function to not be able to deal with `pathlib.Path` objects: [PEP 519](https://www.python.org/dev/peps/pep-0519/).)